packages/core/src/renderTexture/RenderTextureSystem.js

```diff
@@ -141,9 +141,10 @@ export class RenderTextureSystem extends System
      * Erases the render texture and fills the drawing area with a colour
      *
      * @param {number[]} [clearColor] - The color as rgba, default to use the renderer backgroundColor
+     * @param {Number} mask - Bitwise OR of masks that indicate the buffers to be cleared, by default COLOR
```

Comment:
Should be `[mask]` because it’s optional. And type should be `PIXI.BUFFER_BITS` Not Number